src/pages/cart/Cart.jsx

```diff
@@ -1,8 +1,37 @@
 import React from 'react'
-
+import '../cart/cart.css'
+import Navbar from '../components/navbar/Navbar'
+import { useEffect , useState } from 'react'
+import axios from 'axios'
+import CartCard from '../components/card-card/CartCard'
 const Cart = () => {
+  const [localcart , setlocalcart] = useState([])
+  useEffect(()=>{
+    console.log('use effect running');
```

Comment:
Remove unused codes

Reply:
will do. 
as for the credentials, that is a good suggestion. will do that ,as it will give a more realistic feel. thank you .